README.md

```diff
@@ -13,3 +9,53 @@ Make sure that you have the compatible versions of Visual Studio 2019 and .NET C
+
+[Download Free Trial](https://www.syncfusion.com/downloads/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples)
+
+[Pricing](https://www.syncfusion.com/sales/products/blazor?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples)
+
+[Documentation](https://blazor.syncfusion.com/documentation/timepicker/getting-started?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples)
+
+[View Demos](https://blazor.syncfusion.com/demos/timepicker/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples)
+
+[Online example](https://blazor.syncfusion.com/demos/timepicker/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples)
+
+[Video Tutorials](https://www.syncfusion.com/tutorial-videos/blazor/timepicker?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples)
+
+[Knowledge base](https://www.syncfusion.com/kb/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples)
+
+## About Syncfusion Blazor Components
+The Syncfusion's [Blazor components](https://www.syncfusion.com/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples) library offers over 70 UI components to work with Blazor server-side and client-side (Blazor WebAssembly) projects seamlessly. In addition to TimePicker, we provide popular Blazor Components such as [DataGrid](https://www.syncfusion.com/blazor-components/blazor-datagrid?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples),[Charts](https://www.syncfusion.com/blazor-components/blazor-charts?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples), [Scheduler](https://www.syncfusion.com/blazor-components/blazor-scheduler?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples), [Diagram](https://www.syncfusion.com/blazor-components/blazor-diagram?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples), and [Word Processor](https://www.syncfusion.com/blazor-components/blazor-word-processor?utm_source=github&utm_medium=listing&utm_campaign=blazor-timepicker-github-samples).
+
+### About Syncfusion
+
+Founded in 2001 and headquartered in Research Triangle Park, N.C., Syncfusion has more than 28,000+customers and more than 1 million users, including large financial institutions, Fortune 500 companies, and global IT consultancies.
```

Comment:
change this 
![image](https://user-images.githubusercontent.com/61148953/178489109-ea868e11-c2f8-43b4-9644-43550450e47e.png)

Reply:
Changed